tests/test_sync.py

```diff
@@ -307,6 +316,81 @@ def check_s3_dest(dest_blob):
             check_s3_dest(dest_blob)
             self._assert_content_type(dest_blob, src_blob)
 
+@testmode.integration
+class TestSyncDaemonLargeManifests(unittest.TestCase, DSSSyncMixin):
+    @classmethod
+    def setUpClass(cls):
+        cls.app = ThreadedLocalServer()
+        cls.app.start()
+        cls.flask_app = flask.Flask(__name__)
+
+    @classmethod
+    def teardownClass(cls):
+        cls.app.shutdown()
+
+    def setUp(self):
+        dss.Config.set_config(dss.BucketConfig.NORMAL)
```

Comment:
Should this be `dss.BucketConfig.TEST`?

Reply:
This is an integration test of the deployed sync daemon, so it will be performed in the main bucket.